cmd/dashboard.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/buoyantio/linkerd-buoyant/pkg/k8s"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+)
+
+func newCmdDashboard(cfg *config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "dashboard [flags]",
+		Args:  cobra.NoArgs,
+		Short: "Opens the Bcloud Dashboard",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			client, err := k8s.New(cfg.kubeconfig, cfg.kubecontext, cfg.bcloudServer)
+			if err != nil {
+				return err
+			}
```

Comment:
i don't think the `dashboard` command needs to interact with the k8s cluster (or agent). it can simply open the url.